Gems/ScriptCanvas/Code/Include/ScriptCanvas/Translation/GraphToLua.cpp

```diff
@@ -1160,6 +1165,9 @@ namespace ScriptCanvas
         {
             const auto& staticVariableNames = m_model.GetStaticVariablesNames();
             auto& variables = m_model.GetVariables();
+
+            bool isSelfLocalInitialized = false;
```

Comment:
a reference to the local self entity id is only created once per translated function block